coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
I apologise for that. What I should be talking about is passing the aspects not initiating them. This is surely not the right place to toggle this discussion because this is the metaclass and its job is to instantiate the bear by defining its aspects. I realise it will be problematic in these cases.
```
>>> SomeAspectsBear.aspects['detect']
 +    [<aspectclass 'Root.Metadata.CommitMessage.Shortlog.ColonExistence'>]
```
 We do not expect user ever to mention `Root` as an aspect. I am asking to pass that as aspect of the bear __only__ when user doesn't passes any aspect. My point is we can use same bear collection methodology in both the cases if we pass `Root` aspect as the default aspect of any bear.